app/upgrades/v0_3_0/upgrades.go

```diff
@@ -15,11 +17,13 @@ const (
 var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
+	StoreUpgrades:        storetypes.StoreUpgrades{Added: nil, Renamed: nil, Deleted: nil},
 }
 
 func CreateUpgradeHandler(
 	moduleManager *module.Manager,
 	configurator module.Configurator,
+	keepers *keepers.AppKeepers,
```

Comment:
not sure if I like editing previous migrations vs just adding "ignore lint" comment (here and above and in other old files of migrations)

Reply:
What's your concern? Maybe I'm missing something, but I can't see any potential risks associated with this specific change.